arelle/CntlrCmdLine.py

```diff
@@ -318,6 +318,9 @@ def parseAndRun(args):
     parser.add_option("--nocertificatecheck", action="store_true", dest="noCertificateCheck", help=SUPPRESS_HELP)
     parser.add_option("--httpUserAgent", action="store", dest="httpUserAgent", 
                       help=_("Specify non-standard http header User-Agent value"))
+    parser.add_option("--httpsRedirectCache", action="store_true", dest="httpsRedirectCache",
+                      help=_("Check for missing cache files in their inverse http/https cache directory"))
+    parser.add_option("--httpsredirectcache", action="store_true", dest="httpsRedirectCache", help=SUPPRESS_HELP)
```

Comment:
Move this so that it's not splitting up/between the `httpuseragent`/`httpUserAgent` options.

The help text should probably be tweaked. It kind of sounds like http only looks in https and vice versa. How about `Treat http and https schemes interchangeably when looking up files from the webcache`?